dev/devicelab/lib/tasks/web_benchmarks.dart

```diff
@@ -38,7 +38,8 @@ Future<TaskResult> runWebBenchmark(WebBenchmarkOptions benchmarkOptions) async {
       options: <String>[
         'web',
         '--no-tree-shake-icons', // local engine builds are frequently out of sync with the Dart Kernel version
-        if (benchmarkOptions.useWasm) ...<String>['-O4', '--wasm', '--no-strip-wasm'],
```

Comment:
Ah, so this previous `-04` was redundant

Reply:
Yes. It's currently `-O4` by default in profile and release mode.